NEWS.md

```diff
@@ -1,5 +1,43 @@
 # What's new?
 
+## 11.x
+
+Since Neovim 0.10 has been released, I've been working on a new version of **LazyVim**
+that is fully compatible with all the latest Neovim features.
+
+Additionally, some core plugins have been moved to extras.
+
+- `native snippets` are not the default on Neovim 0.10
+  Older versions of Neovim will use the new `luasnip` extra.
+
+- `native comments` are now the default on Neovim 0.10
+  Older versions of Neovim will use the new `mini-comment` extra.
+  `nvim-ts-context-commentstring` has been integrated in the native comments.
+
+- `inlay hints` have been in **LazyVim** for a while, but are now
+  enabled by default. To disable then:
+
+  ```lua
+  {
+    "nvim-lspconfig",
+    opts = {
+      inlay_hints = { enabled = true },
+    }
+  }
+  ```
+
+- plugins moved to extras:
+
+  - `mini.ai` which I couldn't live without, but not everyone needs it
+  - `mini.surround`
+  - `mini.indentscope` scopes are now also highlighted with `indent-blankline`
+  - `nvim-treesitter-context`
+
+- There's a new extra for the `nvim-treesitter` **rewrite**.
+  Since the rewrite is not backward compatible, some plugins will be disabled
+  when you enable this extra: `vim-illuminate`, `nvim-ts-autotag`, and `nvim-ts-autotag`.
+  I would **NOT** recommend enabling this extra for now.
```

Comment:
![image](https://github.com/LazyVim/LazyVim/assets/2361214/598d09fd-e4f1-4085-9990-e7822b122875)

Reply:
Oh I see. Your initial post didn't clarify it was about `nvim-treesitter`, so I assumed you were talking about Neovim 0.11. Sorry about that.

Reply:
I just disabled the `treesitter-rewrite` extra. I added it since some people told me the plan was to release the rewrite soon after 0.10 and I wanted to be ready.

I've been using the rewrite locally and it's been working really well for me. (with the proper fixes 
 for https://github.com/windwp/nvim-ts-autotag/pull/171)

Looking forward to the release!

Reply:
Thank you. I suspect your risk persona is very different from the typical LazyVim user. Especially for Windows users, all bets are off. (I've started getting issues on nvim-treesitter that stemmed from wrong expectations, and cost me quite a bit of time tracking down to LazyVim users running the `main` branch.)

Reply:
ouch, sorry about that :)
Do let me know when the rewrite it ready for more testing, then I can enable the extra for LazyVim again.

Tip: create a separate issue template for the rewrite and add a proper label, so you can easily filter the issues

Reply:
That assumes that people are supposed to be using the rewrite, which is not at all the case. I'm just dogfooding the dev branch.

Reply:
ok, I thought the idea was to make the `main` branch the main branch at some point.

Reply:
Yes, at _some_ point, in the not imminent future ;)

Reply:
Got it. But so if you're at a point where you want more testers, let me know :)

Reply:
(I know the original roadmap tied the switch to Nvim 0.10, but that has not panned out for a variety of reasons. I'm tentatively aiming for 0.11 now, but no promises.)